web/src/lib/types/types.ts

```diff
@@ -463,11 +463,7 @@ export type SidebarCardContent = {
   header?: string;
   notStartedHeader?: string;
   completedHeader?: string;
```

Comment:
pebble, should `preBodySpanButtonText` be added here as well?

Reply:
added